pom.xml

```diff
@@ -228,5 +225,33 @@
                 </plugins>
             </build>
         </profile>
+
+        <!--module activation profiles-->
+        <profile>
+            <id>all-modules</id>
+            <modules>
+                <module>microprofile-health</module>
+                <module>microprofile-open-api</module>
+                <module>microprofile-metrics</module>
+            </modules>
+        </profile>
+        <profile>
+            <id>mp-health</id>
+            <modules>
+                <module>microprofile-health</module>
+            </modules>
+        </profile>
+        <profile>
```

Comment:
keep just this profile and remove all the others

health and metrics viz above
`all-modules` will get easily out of sync
profiles are sort of temporary, all valid tests (for latest released WF) should be executed with "mvn clean verify"

Reply:
Thank you! I didn't understand the temporary nature of these profiles before, will fix.